packages/calcite-components/src/components/panel/panel.e2e.ts

```diff
@@ -360,6 +366,42 @@ describe("calcite-panel", () => {
     expect(calcitePanelClose).toHaveReceivedEventTimes(1);
   });
 
+  it("close event can be cancelled", async () => {
+    const page = await newProgrammaticE2EPage();
+    await page.evaluate(() => {
+      (window as TestPanelWindow).calledTimes = 0;
+
+      const panel = document.createElement("calcite-panel");
+      panel.heading = "Hello World";
+      panel.closable = true;
+      panel.innerText = "Hello World";
+
+      panel.addEventListener("calcitePanelClose", (event) => {
+        event.preventDefault();
+        (window as TestPanelWindow).lastEventCancelable = event.cancelable;
```

Comment:
Can you add a comment explaining how this is needed to work around an event spy limitation (i.e., details are captured before event is canceled)?